stubs/openpyxl/openpyxl/worksheet/datavalidation.pyi

```diff
@@ -81,7 +84,7 @@ class DataValidation(Serialisable):
         allow_blank: Incomplete | None = False,
     ) -> None: ...
     def add(self, cell) -> None: ...
-    def __contains__(self, cell): ...
+    def __contains__(self, cell: _HasCoordinate) -> bool: ...
```

Comment:
This doesn't seem right to me, as a `hasattr` check explicitly checks for the the existence of `coordinate`, but continues whether it exists or not. I don't know what the correct annotation should be though. Maybe just `_HasCoordinate | Incomplete` for now?

Reply:
`_HasCoordinate | CellRange`, thanks for the heads-up. I went too fast here and forgot to compelte the annotation after making the protocol.

Reply:
`_HasCoordinate | CellRange | str`, thanks for the heads-up. I went too fast here and forgot to compelte the annotation after making the protocol.